speech.d.ts

```diff
@@ -21,6 +21,60 @@ declare module 'lmnt-node' {
 
     /** The desired target length of the output speech in seconds. */
     length?: number;
+
+    /** If `True`, the response will include word durations detail. */
+    return_durations?: boolean;
+
+    /** If `True`, the response will include the seed used for synthesis. */
+    return_seed?: boolean;
+  }
+
+  /**
+   * Options available to pass with streaming synthesis requests. See the
+   * [SDK documentation](https://lmnt.com/docs/node/) for available values
+   * and defaults.
+   */
+  interface StreamingSynthesisOptions {
+    /** Whether to return extra data (durations data and warnings) with each audio chunk. */
+    return_extras?: boolean;
+
+    /** The desired speed of the synthesized speech. */
+    speed?: number;
+
+    expressive?: number;
```

Comment:
Add brief description.

Reply:
@zachoverflow does this default to 0.8? I've seen it say that in the REST API synthesize but I'm not sure. Also, do we want to add this to everything now? It's not in `synthesize` for this SDK and not in the Python SDK at all.

Reply:
yes defaults to 0.8. ok to add to docs but not to hardcode in the sdk itself